CRM/Dedupe/BAO/DedupeRuleGroup.php

```diff
@@ -104,7 +104,7 @@ public static function supportedFields($requestedType): array {
         $fields[$ctype]['civicrm_contact']['sort_name'] = ts('Sort Name');
 
         // add all custom data fields including those only for sub_types.
-        foreach (self::getTree($ctype, NULL, NULL, -1, [], NULL, TRUE, NULL, TRUE) as $key => $cg) {
+        foreach (self::getTree($ctype) as $key => $cg) {
```

Comment:
@eileenmcnaughton am I right in thinking this is effectively setting $returnAll to be TRUE instead of FALSE which is why L684 can go right? (just wanted to double check given the number of parameters here)

Reply:
@seamuslee001 yes - that is correct - the only non-default values passed in are contactType, GroupID & returnAll - with the latter 2 being fixed values